### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by alex1779**
# Portfolio Mobile Menu

## Description

For this milestone of my portfolio website, I will be implementing the mobile menu using my recently acquired knowledge of JavaScript.

For this project I have added JavaScript + HTML + CSS  linter's files.

On mobile devices, the following features have been implemented:

-      When the user clicks (or taps) the hamburger button, the mobile menu appears.
-      When the user clicks (or taps) the close (X) button, the mobile menu disappears.
-      When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
-      When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by mirouhml — CHANGES_REQUESTED** [review]
Hi @alex1779, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request, that has no meaningful description, kindly add a description explaining what you did in this project, you also need to customize your README as it doesn't have any information, it's just the template, kindly adjust that as well.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  buttonAbout.style.border = 'none';
+  buttonContact.style['aspect-ratio'] = '251 / 24';
+  buttonContact.style.width = '90%';
+  buttonContact.style.background = 'transparent';
+  buttonContact.style.color = 'white';
+  buttonContact.style['font-size'] = '9.5vw';
+  buttonContact.style['font-weight'] = '600';
+  buttonContact.style['line-height'] = '1.5vw';
+  buttonContact.style['text-align'] = 'left';
+  buttonContact.style.border = 'none';
```
Comment (tugsem) [inline js/index.js]: - [x] A function should be as short as possible. 20 to 30 lines is probably the maximum length of a function. They should only do one thing, so there's no reason for them to be longer than that. Reducing the length makes it easier to read. Please remove these styling related lines and move them to your css file.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+function myFunction2() {
+  document.querySelector('#mobile-menu').style.display = 'none';
+  document.querySelector('#icon-button-button').style.display = 'flex';
+  document.querySelector('#div-logo').style.display = 'flex';
+  document.querySelector('#headline').style.display = 'block';
+}
```
Comment (tugsem) [inline js/index.js]: - [ ]  Please use more descriptive function name here for best practices and also readability. Moreover, you are using the same lines over and over again in your functions below. Kindly create one proper function for this job and kindly create 'click' event listener on mobile menu list items and call this function when dom element is triggered.

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+function contactClick() {
+  document.querySelector('#mobile-menu').style.display = 'none';
+  document.querySelector('#icon-button-button').style.display = 'flex';
+  document.querySelector('#div-logo').style.display = 'flex';
+  document.querySelector('#headline').style.display = 'block';
+  document.querySelector('#contact-form-data-h1').scrollIntoView();
+}
+
+function myFunction() {
```
Comment (tugsem) [inline js/index.js]: - [x] Please use more descriptive function name here for best practices and also readability.

File: js/index.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  buttonPortfolio.onclick = function () { portfolioClick(); };
+  buttonAbout.onclick = function () { aboutClick(); };
+  buttonContact.onclick = function () { contactClick(); };
+  navbar.appendChild(buttonPortfolio);
+  navbar.appendChild(buttonAbout);
+  navbar.appendChild(buttonContact);
+  menu.appendChild(navbar);
+}
+const menuhamb = document.querySelector('#icon-button-button');
+menuhamb.onclick = function () { myFunction(); };
```
Comment (tugsem) [inline js/index.js]: - [x] When I click mobile menu icon, I should see blurry picture of background and menu shouldn't be scrollable. Your menu background doesn't seem like figma and it's scrollable. Also when I try to toggle menu icon, it crashes second time when I open it. You can check the video link below. Kindly fix this. 
https://screen-recorder-bucket.s3.ap-south-1.amazonaws.com/ScreenRecorder_2022-10-18_4fdfbaa9-69a1-4df4-ba37-f6238b51bbc0.mp4

File: README.md [hunk 5]
```diff
@@ -1,8 +1,23 @@
...
+## Project Tasks List (Working-in-progress)
+
+- [x] Setting up the initial project structure
+- [x] mobile version skeleton
+- [x] Contact form
+- [x] Desktop version
+- [x] Application deploy
+- [x] Evaluate accessibility
+- [x] Mobile Menu
+
```
Comment (tugsem) [inline README.md]: - [x] Kindly add these sections filling them as much as possible to help someone wants to clone and run your project on live server:
 
- Getting Started
To get a local copy up and running follow these simple example steps.

- Prerequisites
- Setup
- Install
- Usage

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @alex1779 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!
Alt Text
## Highlights ⭐ 
✔️ Linters
✔️ Readme
✔️ Descriptive pr

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @alex1779, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**